File: app/src/main/java/tk/lorddarthart/rxnewstestapp/News.java
package tk.lorddarthart.rxnewstestapp;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

import org.json.JSONArray;

import java.util.List;

public class News {
    @SerializedName("News")
    @Expose
    private List<Item> news;

    List<Item> getNews() {
        return news;
    }

    public void setNews(List<Item> news) {
        this.news = news;
    }
}

File: app/src/main/java/tk/lorddarthart/rxnewstestapp/application/model/News.kt
package tk.lorddarthart.rxnewstestapp.application.model

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

class News {
    @SerializedName("News")
    @Expose
    internal var news: List<Item>? = null
        set
}
